VMEncryption/main/handle.py

```diff
@@ -1596,9 +1607,14 @@ def find_all_devices_to_encrypt(encryption_marker, disk_util, bek_util, volume_t
             if device_item.mount_point is None or device_item.mount_point == "":
                 # Don't encrypt partitions that are not even mounted
                 continue
-            if os.path.join('/dev/', device_item.name) not in dev_path_reference_table:
+            if os.path.join('/dev/', device_item.name) not in dev_path_reference_table and not device_item.name.startswith(CommonVariables.nvme_disk_identifier):
```

Comment:
risk seems small - but if user happens to have a non-nvme mapper name that also happens to start with "nvme" for some reason - this could misidentify, maybe something to call out in some documentation footnotes